Readme.md

```diff
@@ -373,6 +373,33 @@ OPTIONS:
             Use as `--rustc-extra-args="--my-arg"`
 ```
 
+### Upload
+
+```
+Uploads the distribution files to the repository (package index)
```

Comment:
```suggestion
Uploads python packages to pypi
```

The term "distribution" is not clear to most people, so I'm trying to use "python packages" everywhere. I think it makes sense here to say pypi since it is the default, while everyone running their own package index will still find `-r`.

Reply:
Applied